Makefile

```diff
@@ -23,6 +23,9 @@ parser_cargo_args = --release --manifest-path parser/Cargo.toml --target x86_64-
 $(parser): parser
 	cargo build $(parser_cargo_args)
 
+# TODO: talk to Phil about make again
```

Comment:
nit: still TODO ?

Reply:
Yes, it was. Good catch. I just spoke with him about this. I wasn't sure about using `parser_cli` as an "alias task" of sorts.

As we spoke, we decided that since CI is handling running _all_ of the things entirely outside the Makefile, and running any _one_ of the tasks defined within was fairly easy "docker build" or "go test", the Makefile was probably no longer necessary. We can always bring it back if we want more coordination.